examples/language/performance_evaluator.py

```diff
@@ -113,13 +113,13 @@ def on_step_start(self, step: int) -> None:
         self.disable = self.ignore_steps > 0 and step < self.ignore_steps
         if self.disable:
             return
-        get_accelerator().synchronize()
+        # get_accelerator().synchronize()
```

Comment:
Why comment this line?

Reply:
There is async communication between `optimizer.step()` and `model(**data)`. Synchronization may make it ineffective.